ChatWithAuthSample/AuthorizationSample/Startup.cs

```diff
@@ -0,0 +1,177 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Authentication.JwtBearer;
+using Microsoft.AspNetCore.Builder;
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.AspNetCore.HttpsPolicy;
+using Microsoft.AspNetCore.Identity;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Options;
+using Microsoft.IdentityModel.Tokens;
+
+namespace AuthorizationSample
+{
+    public class Startup
+    {
+        public Startup(IConfiguration configuration)
+        {
+            Configuration = configuration;
+        }
+
+        public IConfiguration Configuration { get; }
+
+        // This method gets called by the runtime. Use this method to add services to the container.
+        public void ConfigureServices(IServiceCollection services)
+        {
+            // Configurate Database Context for dependency injection.
+            services.AddDbContext<Persistence.UserContext>(options =>
+            {
+                options.UseSqlServer(Configuration.GetConnectionString("DefaultConnection"));
+            });
+
+            // Add SignalR Services.
+            services.AddSignalR();
+
+            // Configure our Identity Services.
+            this.ConfigureIdentity(services);
+
+            services.AddMvc().SetCompatibilityVersion(CompatibilityVersion.Version_2_1);
+
+            // Configure Authz
+            this.ConfigureAuthentication(services);
+        }
+
+        /// <summary>
+        /// Configures Identity Services and Contexts
+        /// </summary>
+        /// <param name="services"></param>
+        public void ConfigureIdentity(IServiceCollection services)
+        {
+            services.AddIdentity<IdentityUser, IdentityRole>()
+                .AddDefaultTokenProviders()
+                .AddEntityFrameworkStores<Persistence.UserContext>();
+
+            services.Configure<IdentityOptions>(options =>
```

Comment:
really comprehensive way to mix identity in. this is a great example.

Reply:
Thank you for your kind words, it is appreciated. I'm in the process of developing a very complex SignalR Core based app at the moment and decided to distill down some of the stuff I learned along the way. I'm going to be likely adding an Angular 6+ client repo for this sample this weekend as there seems to be interest in the issue tracker. I'm more than willing to add that into the sample work if you would like.